src/Tree.tsx

```diff
@@ -286,7 +286,7 @@ const generateNewTree = <
   let tempRecord: NodeRecord<TNodePublicState> | null = rootRecord;
 
   const useIdleCallback =
-    'requestIdleCallback' in window &&
+    typeof requestIdleCallback !== 'undefined'
```

Comment:
You forgot to add `&&` to the condition
```suggestion
    typeof requestIdleCallback !== 'undefined' &&
```